packages/common/src/access/can-edit-item.ts

```diff
@@ -0,0 +1,46 @@
+import { IItem, IUser, IGroup } from "@esri/arcgis-rest-types";
+import { isUpdateGroup, includes } from "../utils";
+import { getProp } from "../objects";
+import { hasBasePriv } from "./has-base-priv";
+
+/**
+ *
+ * @param {IItem} model
+ * @param {IUser} currentUser
+ * @returns {boolean}
+ */
+export function canEditItem(model: IItem, currentUser?: IUser): boolean {
+  let res = false;
+  if (currentUser) {
+    const itemControls = ["admin", "update"];
+    const { itemControl, owner, orgId: itemOrgId } = model;
+    let { isOrgItem } = model;
```

Comment:
is `isOrgItem` a thing we stab on items sometimes? If so, and this is truly needed, should pass this in via an options argument instead of relying on consumers stabbing an undoc'd prop onto the item?

Reply:
`isOrgItem` comes from portal API, but i could see it being redundant to just checking item.orgId against user.orgId